docs/content/users/usage/faq.md

```diff
@@ -17,6 +17,15 @@ DDEV works nearly anywhere Docker will run, including macOS, Windows 10/11 Pro/E
 * It’s focused directly on running containers.
 * It’s fast and stable.
 
+### How can I migrate from one Docker provider to another?
+
+There are many Docker providers on the various platforms supported by DDEV. For example, on macOS people use Docker Desktop and Colima (both officially supported and with automated tests) and they also use [OrbStack](https://orbstack.dev/) and [Rancher Desktop](https://rancherdesktop.io/), which don't yet have official DDEV support or automated tests. On Windows WSL2, people may use Docker Desktop or Docker CE inside WSL2. In all cases, if you want to switch between Docker providers, save your database and make sure the two Docker providers don't interfere with each other, so follow these steps:
+
+1. Save away your projects' databases. An easy way to do this is `ddev snapshot --all`, which will make snapshots of all *registered* projects (projects that show up in `ddev list`). Of course if you have dumps of your databases using other techniques, you don't have to do this.
+2. Stop the Docker provider you're moving from. For example, exit Docker Desktop.
+3. Start the Docker provider you're moving to.
+4. Start projects and restore their databases, for example `ddev snapshot restore --latest`.
```

Comment:
Keep that perfect short sentence flow with a period, and run the command. Also attempted to connect `snapshot restore` to the first step for completeness, but you decide whether that’s working.

```suggestion
4. Start projects and restore their databases. For example, you could run `ddev snapshot restore --latest` to load a snapshot taken in step one.
```